test/notebook_test_runner/test_notebooks_in_dss_docker_image.py

```diff
@@ -81,7 +81,7 @@ def test_notebook(notebook_test_container_with_log, notebook_test_file, notebook
     virtual_env = exec_command(command_echo_virtual_env, container)
     command_run_test = (
         f"{virtual_env}/bin/python"
-        f" -m pytest --setup-show -s --backend={notebook_test_backend} {notebook_test_file}"
+        f" -m pytest --setup-show -s --backend={notebook_test_backend} --itde-nameserver='8.8.8.8' --{notebook_test_file}"
```

Comment:
Not sure if this is the correct way

Reply:
Ok, according to the tests this should be ok:
https://github.com/exasol/pytest-plugins/blob/f158045b3045f9065d9d8ce0c16aad3f3806d941/pytest-backend/test/integration/pytest_backend_test.py#L67